core/HoistAppModel.ts

```diff
@@ -52,6 +52,11 @@ export class HoistAppModel extends HoistModel {
      */
     async logoutAsync() {}
 
+    /**
+     * Allow for app-specific support of version bar
+     */
+    get supportVersionBar(): boolean {return true}
```

Comment:
lets call this supportsVersionBar(), and also improve the comment to something a bit more descriptive, e.g.  `Should the version bar be shown in this application?.   Applications  for which  a version bar might not be appropriate (e.g. a mini- app being shown in a frame or modal) may override this getter and return false`